### ISSUE TITLE
The ability to share groups

### ISSUE THREAD

**Issue by anubis2814** [friendica/friendica#10043]
One of the few reasons I still stick around Mewe is their groups.  I just enjoy having a ready-made feed of pretty pictures or topic related posts. I've made groups for myself but if I could share them, the only reason to still be on Mewe would be gone.  I could draw Mewe users over here, and most Mewe users I know are passively trying to figure out a facebook alternative but haven't had much luck but still enjoy the groups ability of Mewe.  It should also be shareable with mastodon as lists and maybe even twitter depending on the protocol.  Also ensure its a one time copy so that the group is now theirs, and they can modify it as they see fit.  Sharing with fellow friendica users should be relatively easy don't know how easy that will be with other platforms though.  Thanks!

**Comment by anubis2814** [comment]
PS I looked and Twitter can share lists via a link, Mastodon cannot.  Maybe I should make a recommendation over there too.

**Comment by MrPetovan** [comment]
When you mean "group", do you mean "a list of follows under an arbitrary name"?

**Comment by anubis2814** [comment]
Yes, not to be confused with a forum which is what facebook calls a group

**Comment by MrPetovan** [comment]
Thank you for the elaboration, this detail about Facebook nomenclature is precisely why I asked.
